Gifski/MainWindowController.swift

```diff
@@ -61,6 +62,10 @@ final class MainWindowController: NSWindowController {
 		}
 	}
 
+	var conversionCompleted: Bool {
```

Comment:
```suggestion
	var isConversionCompleted: Bool {
```

Reply:
Damn, not sure what my brain was doing here, but of course it should be `isConversionCompleted`, good catch.